### PR TITLE
TL Localization - Chapter 6, section 3 - part 2 (1,599 words)

### PR DESCRIPTION
**Pull request by heywens**

### CODE REVIEW

File: book/06-github/sections/3-maintaining.asc [hunk 1]
```diff
@@ -218,160 +218,160 @@ Mayroon ding ref na `refs/pull/#/merge` sa panig ng GitHub , na kumakatawan sa c
...
+Sa katunayan, maaari ka ring mag-target sa ibang Kahilingan na Pull.
 
-If you see a Pull Request that is moving in the right direction and you have an idea for a change that depends on it or you're not sure is a good idea, or you just don't have push access to the target branch, you can open a Pull Request directly to it.
+Kung nakikita mo ang isang Kahilingan na Pull na gumagalaw sa tamang direksyon at mayroon kang ideya para sa isang pagbabago na nakadepende dito o hindi ka sigurado na magandang ideya, o wala ka lamang access sa pag-push sa branch ng target, maaari kang magbukas ng isang Kahilingan na Pull na direkta dito.
 
-When you go to open a Pull Request, there is a box at the top of the page that specifies which branch you're requesting to pull to and which you're requesting to pull from.
-If you hit the ``Edit'' button at the right of that box you can change not only the branches but also which fork.
+Kapag ikaw ay nagbubukas ng isang Kahilingan na Pull, may isang kahon sa itaas ng pahina na tumutukoy kung aling branch ang hinihingan mo na maka-pull at kung saan hinihiling mo na mag-pull.
```
Comment (mind-blast) [inline book/06-github/sections/3-maintaining.asc]: Kapag ikaw ay nagbubukas to Kapag ikaw ay pumunta upang bumukas ng

File: book/06-github/sections/3-maintaining.asc [hunk 2]
```diff
@@ -218,160 +218,160 @@ Mayroon ding ref na `refs/pull/#/merge` sa panig ng GitHub , na kumakatawan sa c
...
+Kung pindutin mo ang ``Edit'' na pindutan sa kanan ng kahon na iyon, maaari kang magbago hindi lamang sa mga branch kundi rin sa kung anong fork.
```
Comment (mind-blast) [inline book/06-github/sections/3-maintaining.asc]: Kung pindutin mo ang to Kung pinindot mo ang

File: book/06-github/sections/3-maintaining.asc [hunk 3]
```diff
@@ -218,160 +218,160 @@ Mayroon ding ref na `refs/pull/#/merge` sa panig ng GitHub , na kumakatawan sa c
...
 
 [[_pr_targets]]
-.Manually change the Pull Request target fork and branch.
-image::images/maint-04-target.png[PR targets]
+.Baguhin nang mano-mano ang fork na target ng Kahilingan na Pull at branch.
```
Comment (mind-blast) [inline book/06-github/sections/3-maintaining.asc]: Baguhin nang mano-mano ang fork na target ng Kahilingan na Pull at branch.  to  Mano-manong baguhin ang Kahilingan ng Pull na target ang fork at branch

File: book/06-github/sections/3-maintaining.asc [hunk 4]
```diff
@@ -218,160 +218,160 @@ Mayroon ding ref na `refs/pull/#/merge` sa panig ng GitHub , na kumakatawan sa c
...
+image::images/maint-04-target.png[Mga target ng PR]
 
-Here you can fairly easily specify to merge your new branch into another Pull Request or another fork of the project.
+Dito maaari kang medyo madaling magtukoy upang pagsamahin ang iyong bagong branch sa ibang Kahilingan na Pull o ibang fork ng proyekto.
 
-==== Mentions and Notifications
+==== Mga Pagbanggit at Abiso
```
Comment (mind-blast) [inline book/06-github/sections/3-maintaining.asc]: Mga Pagbanggit at mga Abiso

File: book/06-github/sections/3-maintaining.asc [hunk 5]
```diff
@@ -218,160 +218,160 @@ Mayroon ding ref na `refs/pull/#/merge` sa panig ng GitHub , na kumakatawan sa c
...
 
-GitHub also has a pretty nice notifications system built in that can come in handy when you have questions or need feedback from specific individuals or teams.
+Mayroon ding magandang built on na sistema ng mga abiso ang GitHub na magagamit kapag ikaw ay may mga tanong o kailangan na katugunan mula sa tiyak na mga indibidwal o koponan.
```
Comment (mind-blast) [inline book/06-github/sections/3-maintaining.asc]: change built on to built in

File: book/06-github/sections/3-maintaining.asc [hunk 6]
```diff
@@ -218,160 +218,160 @@ Mayroon ding ref na `refs/pull/#/merge` sa panig ng GitHub , na kumakatawan sa c
...
 
-In any comment you can start typing a `@` character and it will begin to autocomplete with the names and usernames of people who are collaborators or contributors in the project.
+Sa anumang komento maaari mong simulan sa pagtitipa ng karakter na `@` at ito ay magsisimula sa autocomplete kasabay ng mga pangalan at username ng tao na mga tagatulong o nag-aambag sa proyekto.
```
Comment (mind-blast) [inline book/06-github/sections/3-maintaining.asc]: change autocomplete  to awtomatikong matapos

File: book/06-github/sections/3-maintaining.asc [hunk 7]
```diff
@@ -218,160 +218,160 @@ Mayroon ding ref na `refs/pull/#/merge` sa panig ng GitHub , na kumakatawan sa c
...
 
-.Start typing @ to mention someone.
-image::images/maint-05-mentions.png[Mentions]
+.Simulang magtipa ng @ upang banggitin ang isang tao.
```
Comment (mind-blast) [inline book/06-github/sections/3-maintaining.asc]: change Simulang magtipa ng to Simulan ang pag-type

File: book/06-github/sections/3-maintaining.asc [hunk 8]
```diff
@@ -218,160 +218,160 @@ Mayroon ding ref na `refs/pull/#/merge` sa panig ng GitHub , na kumakatawan sa c
...
+image::images/maint-05-mentions.png[Mga Pagbanggit]
 
-You can also mention a user who is not in that dropdown, but often the autocompleter can make it faster.
+Maaari ka ring magbanggit ng isang gumagamit na wala sa dropdown, ngunit kadalasang maaaring mapabilis ng autocompleter.
 
-Once you post a comment with a user mention, that user will be notified.
-This means that this can be a really effective way of pulling people into conversations rather than making them poll.
-Very often in  Pull Requests on GitHub people will pull in other people on their teams or in their company to review an Issue or Pull Request.
+Sa sandaling magpaskil ka ng komento sa isang pagbanggit ng user, aabisuhan ang user na iyon.
```
Comment (mind-blast) [inline book/06-github/sections/3-maintaining.asc:248]: change magpaskil to mag-post

File: book/06-github/sections/3-maintaining.asc [hunk 9]
```diff
@@ -218,160 +218,160 @@ Mayroon ding ref na `refs/pull/#/merge` sa panig ng GitHub , na kumakatawan sa c
...
+Kadalasan sa mga Kahilingan na Pull sa mga tao sa GitHub ay magpu-pull ng iba pang mga tao sa kanilang mga koponan o sa kanilang kompanya upang magsuri ng isang Isyu o Kahilingan na Pull.
 
-If someone gets mentioned on a Pull Request or Issue, they will be ``subscribed'' to it and will continue getting notifications any time some activity occurs on it.
-You will also be subscribed to something if you opened it, if you're watching the repository or if you comment on something.
-If you no longer wish to receive notifications, there is an ``Unsubscribe'' button on the page you can click to stop receiving updates on it.
+Kung may isang tao ang nakuhang nabanggit sa isang Kahilingan na Pull o Isyu, sila ay magiging ``naka-subscribe'' nito at patuloy na makakakuha ng mga abiso sa anumang oras na nangyayari ang aktibidad nito.
+Ma-subscribe ka rin sa isang bagay kung binuksan mo ito, kung nanonood ka sa repositoryo o kung ikaw ay nagkomento ka sa isang bagay.
+Kung hindi mo na nais na makatanggap ng mga abiso, may isang pindutan na ``Unsubscribe'' sa pahina na maaari mong i-click upang ihinto ang pagtanggap ng mga update nito.
```
Comment (mind-blast) [inline book/06-github/sections/3-maintaining.asc]: change update  to pagbago

File: book/06-github/sections/3-maintaining.asc [hunk 10]
```diff
@@ -218,160 +218,160 @@ Mayroon ding ref na `refs/pull/#/merge` sa panig ng GitHub , na kumakatawan sa c
...
-If you have this turned on you will get emails for each notification.
-We saw examples of this in <<_email_notification>> and <<_email_pr>>.
-The emails will also be threaded properly, which is nice if you're using a threading email client.
+Ang mga abiso sa Email ay ang ibang paraan na mapangasiwaan mo ang mga abiso sa GitHub.
+Kung ini-off mo ito makakakuha ka ng mga email sa bawat abiso.
+Nakikita natin ang mga halimbawa nito sa <<_email_notification>> at <<_email_pr>>.
+Ang mga email ay ma-thread din nang mabuti, na maganda kung ika'y gumagamit ng isang kliyente ng pag-thread ng email.
 
-There is also a fair amount of metadata embedded in the headers of the emails that GitHub sends you, which can be really helpful for setting up custom filters and rules.
+Mayroon ding isang makatarungang dami ng metadata na naka-embed sa mga header ng mga email na ipinadala sa iyo ng GitHub, na maaaring maging kapaki-pakinabang para sa pag-set up ng mga pasadyang filter at patakaran.
```
Comment (mind-blast) [inline book/06-github/sections/3-maintaining.asc:293]: naka-embed to nakadikit

File: book/06-github/sections/3-maintaining.asc [hunk 11]
```diff
@@ -218,160 +218,160 @@ Mayroon ding ref na `refs/pull/#/merge` sa panig ng GitHub , na kumakatawan sa c
...
 
-There are a couple of special files that GitHub will notice if they are present in your repository.
+Mayroong ilang mga espesyal na file na mapapansin ng GitHub kung naroon sila sa iyong repositoryo.
 
 ==== README
 
-The first is the `README` file, which can be of nearly any format that GitHub recognizes as prose.
-For example, it could be `README`, `README.md`, `README.asciidoc`, etc.
-If GitHub sees a README file in your source, it will render it on the landing page of the project.
+Ang una ay ang file na `README`, na maaaring maging halos sa anumang format na nakikilala ng GitHub bilang prosa.
```
Comment (mind-blast) [inline book/06-github/sections/3-maintaining.asc:325]: change prosa to tuluyan

**Comment by mind-blast** [comment]
Please change suggested words . Thank you.

File: book/06-github/sections/3-maintaining.asc (hunk 11 above)
Comment (heywens) [reply book/06-github/sections/3-maintaining.asc:325]: hi sir @robin-ho, prose can also be translated to `prosa`.
references: 
* https://translate.google.com.ph/#en/tl/prose
* https://glosbe.com/en/tl/prose

**Comment by heywens** [comment]
Hi Sir @robin-ho, changes have been applied.

Few suggestions were not applied though. 
These are the following:
* post (as a verb) can be translated also to `magpaskil` - https://translate.google.com.ph/#en/tl/post
* embed (verb) > i want to retain it as is but since it's used as a verb in the sentence, i've translated it as `naka-embed`

**Pull request merged**